src/embeddedFrontend/resultView/resultView.ts

```diff
@@ -501,6 +503,10 @@ export class ResultView {
             return null;
         }
 
+        if (json.__SPECTOR_Object_SOURCE) {
+            return json.__SPECTOR_Object_SOURCE;
+        }
```

Comment:
I don't think this ever did anything. I'd like to display the origin somewhere in the command list detail pane, too